Expensify/App
# Add wallet balance transfer Page

Please review @marcaaron 

### Details


### Fixed Issues

$ https://github.com/Expensify/App/issues/3922

### Tests | QA Steps
1. Open Payments page.
2. Click on the wallet transfer.
3. You should be taken to the Transfer balance page.
4. If your wallet balance is greater than the fee and which account? has an account then you should be able to click the transfer button and it should be enabled.
5. Observe if **Which account?** is a debit card then the Instant method is selected.
6. Observe if **Which account?** is a bank account then the 1-3 business days method is selected.
7. Paypal should not be shown as **Which account?**

### Tested On


### Screenshots


#### Web | Mobile Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/147125901-baa85e55-4348-4dae-9d5e-ffe8b1e723b3.png)

#### iOS


#### Android
